Formula/spaceship.rb

```diff
@@ -0,0 +1,21 @@
+class Spaceship < Formula
```

Comment:
Should this be `Pure`, and not `Spaceship`?

Reply:
> Should this be `Pure`, and not `Spaceship`?

Not sure what happened there! Something going haywire between Atom and Gitkraken…